ext4: add missing ext4_journal_stop()
From: Akinobu Mita <akinobu.mita@gmail.com>

Add missing ext4_journal_stop() in error handling.

Signed-off-by: Akinobu Mita <akinobu.mita@gmail.com>
Cc: Stephen Tweedie <sct@redhat.com>
Cc: adilger@clusterfs.com
Cc: Andrew Morton <akpm@linux-foundation.org>
Cc: Mingming Cao <cmm@us.ibm.com>
Cc: Theodore Tso <tytso@MIT.EDU>
---
 fs/ext4/resize.c |    1 +
 1 file changed, 1 insertion(+)

Index: linux-2.6.25-rc2/fs/ext4/resize.c
===================================================================
--- linux-2.6.25-rc2.orig/fs/ext4/resize.c	2008-02-18 23:50:05.000000000 -0800
+++ linux-2.6.25-rc2/fs/ext4/resize.c	2008-02-18 23:51:23.000000000 -0800
@@ -1035,6 +1035,7 @@ int ext4_group_extend(struct super_block
 		ext4_warning(sb, __FUNCTION__,
 			     "multiple resizers run on filesystem!");
 		unlock_super(sb);
+		ext4_journal_stop(handle);
 		err = -EBUSY;
 		goto exit_put;
 	}
